src/compiler/crystal/command/format.cr

```diff
@@ -68,7 +68,7 @@ class Crystal::Command
           when .invalid_byte_sequence?
             error "'#{result.filename}' is not a valid Crystal source file", exit_code: nil
           when .bug?
-            error "there's a bug formatting '#{result.filename}', please report it including the contents of the file: https://github.com/crystal-lang/crystal/issues", exit_code: nil
+            error "there's a bug formatting '#{result.filename}', to show more information, please run:\n\n  $ crystal tool format '#{result.filename}'", exit_code: nil
```

Comment:
- `FormatResult` does not keep an error object currently. I prefer small fix.
  - It is hard to read multiple backtrace, and each of errors is not related to others. I expect to report formatter error with this backtrace.